README.md

```diff
@@ -1,184 +1,101 @@
-| :------------------------------------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------: |
-| [<img src="https://avatars.githubusercontent.com/u/69719954?v=4" width = "200" />](https://github.com/VAIBHAVIBALAR) | [<img src="https://avatars.githubusercontent.com/u/70349084?s=400&u=17cc870f767d4a220ee656aa5930e609b8166d0d&v=4" width = "200" />](https://github.com/tfbielawski) | [<img src="https://avatars.githubusercontent.com/u/84294481?v=4" width = "200" />](https://github.com/SethBlakley) | [<img src="https://avatars.githubusercontent.com/u/81526595?v=4" width = "200" />](https://github.com/iwood55) | [<img src="https://avatars.githubusercontent.com/u/84815843?v=4" width = "200" />](https://github.com/Broveku) |
-|              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/VAIBHAVIBALAR)              |                                      [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/tfbielawski)                                       |              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/SethBlakley)              |              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/iwood55)              |              [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/Broveku)              |
-|                                                        UX/UI                                                         |                                                                                UX/UI                                                                                |                                                        TPM                                                         |                                                   Front End                                                    |                                                   Front End                                                    |
```

Comment:
It is much easier to focus on the ReadMe without all the contributors listed inside. Thank you for cleaning this up.